scripts/sync.py

```diff
@@ -102,6 +102,8 @@ def main():
                              'server side')
     parser.add_argument('-ds', '--daily_summary', action='store_true',
                         help='Enable daily summary email function')
+    parser.add_argument('-cs', '--check_source', action='store_true',
+                        help='Enable check source email function')
```

Comment:
The help message is unclear. I am unable to propose an alternative because I am not sure what the underlying function does.

Reply:
It goes through existing files on the data sources like `box`, `xnat`, `mediaflux` etc. and checks if the file conforms to the AMP-SCZ SOP file naming. If any file deviates from the SOP pattern, these out of SOP files would be curated in a table and sent to the networks by email.